services/workflows-service/src/business/business.controller.external.ts

```diff
@@ -99,6 +101,21 @@ export class BusinessControllerExternal {
     });
   }
 
+  @UseKeyAuthGuard()
+  @common.Get('/business-information/:registrationNumber')
+  async getCompanyInfo(
+    @common.Query() query: BusinessInformation,
+    @common.Param('registrationNumber') registrationNumber: string,
+  ) {
+    const { jurisdictionCode, vendor } = query;
```

Comment:
vendor can be left to the backend to decide.

both reg number and jurisdiction can be query params